src/Compilers/CSharp/Portable/Symbols/Metadata/PE/TupleTypeDecoder.cs

```diff
@@ -113,7 +113,7 @@ private TupleTypeDecoder(ImmutableArray<string> elementNames)
             TypeSymbol decoded = DecodeTupleTypesInternal(type, elementNames, hasTupleElementNamesAttribute);
             return (object)decoded == (object)type ?
                 metadataType :
-                TypeSymbolWithAnnotations.Create(decoded, nonNullTypes: metadataType.NonNullTypes, isAnnotated: metadataType.IsAnnotated, metadataType.CustomModifiers);
+                TypeSymbolWithAnnotations.Create(decoded, metadataType.NonNullTypesContext, isAnnotated: metadataType.IsAnnotated, metadataType.CustomModifiers);
```

Comment:
Consider keeping the named argument style for nonNullTypesContext

Reply:
Whenever the argument is `this` or `type`, I added the named argument. Whenever the argument is `x.NonNullTypesContext`, I felt the named argument was redundant. 
Also, since we're using a specific type (`INonNullTypesContext`) rather than something like a `bool` or `int`, there is little ambiguity or chance of passing to as wrong parameter.

Reply:
That makes sense.